board/opentrons/ot2/rootfs-overlay/etc/systemd/system/opentrons-status-leds.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Opentrons: Run status LEDs
+Before=opentrons-robot-server.service
+
+[Service]
+Type=notify
+ExecStart=/usr/bin/ot-status-leds
+Restart=on-failure
+RestartSec=10s
+RestartPreventExitStatus=10
+
+[Install]
+WantedBy=basic.target
```

Comment:
@sanni-t In addition to this `WantedBy`, do we also need to manually symlink this file into `/etc/systemd/system/basic.target.wants`?

I'm playing around now with unrelated systemd service stuff. I could be wrong, but it *seems* like specifying `WantedBy=basic.target` doesn't actually automatically pull in the service to `basic.target`.

Instead, it merely specifies that when you run `systemctl enable opentrons-status-leds.service`, systemd will automatically create a symlink to the service in `/etc/systemd/system/basic.target.wants`. And *that* will automatically pull in the service to `basic.target`. Or something. Except that `systemctl enable` doesn't work for us, because `/etc` is read-only. So... 🤷 

Can that be right? To preconfigure interdependent systemd services in buildroot, do we manually need to keep these symlinks in sync?

Reply:
Yep, we add in symlinks to enable systemd services. Further work for enabling & improving this service is in the refresh build branch- https://github.com/Opentrons/buildroot/compare/update_kernel_and_rpi_firmware